languages/en.ini

```diff
@@ -622,6 +622,7 @@ hold_edit_title = "Change Hold Information"
 hold_empty_selection = "No holds were selected"
 hold_error_age_restricted = "A hold cannot be placed due to age restriction on the material."
 hold_error_blocked = "You do not have sufficient privileges to place a hold on this item"
+hold_error_current_loan_patron_group = "This item cannot currently be requested."
```

Comment:
I imagine the institution could customize this with an actual suggestion, like a link to some external request system.